### ISSUE TITLE
Subcommand to verify login status

### ISSUE THREAD

**Issue by weichuliu** [Keeper-Security/Commander#1071]
Hi,

I am trying to find a command to understand the current `keeper` login status.

Likely `keeper (some commands)` returns 0 when the cli is logged in, and returns 1 otherwise.

However, when logged out, `keeper` with any subcommand will get into interactive mode and try to ask for `User(Email):`. So the whole script get stuck.

Is there a way to tell whether keeper is logged in without the cli attempting to wait for user input?

**Comment by weichuliu** [comment]
> Once you logged in the last username is stored into the config file config.json
> You should not be asked for username.

Yes that is true. But I do want my script able to handle the situation when `config.json` does not exist i.e. the user haven't ever login.

But now if the `config.json` is not there, any `keeper` commands will pop a `User(Email)` and got stuck forever, which is not desirable.

**Comment by sk-keeper** [comment]
The login flow can prompt for user input up to 3 times
* device approval 
* two factor authentication
* password
It is not possible to use Commander in unattended mode in all situations.
Keeper provides another API for services. 
https://docs.keeper.io/secrets-manager/secrets-manager/overview

**Comment by ebarrere** [comment]
We pay for KSM, but I would also very much like this functionality. There are situations where it simply does not apply, or using it would actually _reduce_ overall security.

For instance, I would like the ability to pull arbitrary passwords from my vault to pass in to other scripts and systems. Using KSM for this would make _every password_ in my vault accessible to anyone with read-access to the configuration file.

The Commander CLI already provides exactly what I need, **with the major exception of login flow**.

**Comment by ebarrere** [comment]
@maksimu — thank you for the reply.

I understand that, but my point again is that KSM does not apply to my use cases (as you said it is more for machine-to-machine communication) yet the previous comment seems to imply this is the solution:

>   Keeper provides another API for services.
>   https://docs.keeper.io/secrets-manager/secrets-manager/overview

I would very much like the ability to use Commander to automate _my own_ personal workflows, for which KSM does not apply.

**Comment by lancepants** [comment]
This feature would be very useful for the various local scripts that users use to perform daily tasks which require password retrieval.

**Comment by pvagare-ks** [comment]
This feature is available in **Commander v17.1.9** and onwards.

You can use the `keeper login-status` command **outside the Keeper shell** to check the login state:

- If **persistent login** is enabled, it will return `Logged in`
- If not, it will return `Not logged in` without prompting for a password.

here is [documentation](https://docs.keeper.io/en/keeperpam/commander-cli/command-reference/misc-commands#login-status-command) for `keeper login-status` command
